src/main/java/com/google/sps/JobStoreCenter.java

```diff
@@ -109,37 +112,159 @@ private void addJobToDatastore(RunningJob job, Entity project) {
-        String sdkSupportStatus = (String) entity.getProperty("sdkSupportStatus");
-        String region = (String) entity.getProperty("region");
-        int currentWorkers = ((Long) entity.getProperty("currentWorkers")).intValue();
-        String startTime = (String) entity.getProperty("startTime");
-        Double totalVCPUTime = (Double) entity.getProperty("totalVCPUTime"); 
-        Double totalMemoryTime = (Double) entity.getProperty("totalMemoryTime"); 
-        Double totalDiskTimeHDD = (Double) entity.getProperty("totalDiskTimeHDD");
-        Double totalDiskTimeSSD = (Double) entity.getProperty("totalDiskTimeSSD");
-        Long longCurrentVcouCount = (Long) entity.getProperty("currentVcpuCount");
-        Integer currentVcpuCount = longCurrentVcouCount == null? null : longCurrentVcouCount.intValue();
-        Double totalStreamingData = (Double) entity.getProperty("totalStreamingData"); 
-        Boolean enableStreamingEngine = (Boolean) entity.getProperty("enableStreamingEngine");
-        String metricTime = (String) entity.getProperty("metricTime");
-        String state = (String) entity.getProperty("state");
-        String stateTime = (String) entity.getProperty("stateTime");
-        Double price = (Double) entity.getProperty("price");
-        String sdkName = (String) entity.getProperty("sdkName");
-
-        JobJSON job = new JobJSON(projectId, name, id, type, sdk, sdkSupportStatus, region, 
-                                     currentWorkers, startTime, totalVCPUTime, totalMemoryTime,
-                                        totalDiskTimeHDD, totalDiskTimeSSD, currentVcpuCount,
-                                            totalStreamingData, enableStreamingEngine, metricTime,
-                                                state, stateTime, price, sdkName);
-        
+        JobJSON job = convertEntityToJobJSON(entity);
+    
         jobs.add(job);
       }
 
       return jobs;
   }
+
+  private JobJSON convertEntityToJobJSON(Entity entity) {
+    String projectId = (String) entity.getProperty("projectId");
+    String name = (String) entity.getProperty("name");
+    String id = (String) entity.getProperty("jobId");
+    String type = (String) entity.getProperty("type");
+    String sdk = (String) entity.getProperty("sdk");
+    String sdkSupportStatus = (String) entity.getProperty("sdkSupportStatus");
+    String region = (String) entity.getProperty("region");
+    int currentWorkers = ((Long) entity.getProperty("currentWorkers")).intValue();
+    String startTime = (String) entity.getProperty("startTime");
+    Double totalVCPUTime = (Double) entity.getProperty("totalVCPUTime"); 
+    Double totalMemoryTime = (Double) entity.getProperty("totalMemoryTime"); 
+    Double totalDiskTimeHDD = (Double) entity.getProperty("totalDiskTimeHDD");
+    Double totalDiskTimeSSD = (Double) entity.getProperty("totalDiskTimeSSD");
+    Long longCurrentVcouCount = (Long) entity.getProperty("currentVcpuCount");
```

Comment:
Should longCurrentVcouCount be longCurrentVcpuCount or was this a deliberate choice?

Reply:
It was a spelling mistake. Thanks for the heads-up!

Reply:
Fixed here https://github.com/googleinterns/step229-2020/pull/51/commits/170f36f5dab55c3e0ab89771c1d333da18599053